Build errors

Hello,
I get the following error "} expected" in many files such as SessionDto, AddSessionDto, AddTrackDto, .... 
i solved it by replacing the semicolon in line of namespace  with curly brackets  
after solving these issues i get another errors in the web project 

EventHub.Admin.Web\Pages\EventManagement.razor.cs(47,39,47,57): error CS0115: 'EventManagement.OnInitializedAsync()': no suitable method found to override

EventHub.Admin.Web\Pages\OrganizationManagement.razor.cs(47,39,47,57): error CS0115: 'OrganizationManagement.OnInitializedAsync()': no suitable method found to override

EventHub.Admin.Web\Pages\OrganizationMembershipManagement.razor.cs(37,39,37,57): error CS0115: 'OrganizationMembershipManagement.OnInitializedAsync()': no suitable method found to override

EventHub.Admin.Web\Components\UserPicker\UserPicker.razor.cs(56,39,56,59): error CS0115: 'UserPicker.OnParametersSetAsync()': no suitable method found to override

EventHub.Admin.Web\Components\UserPicker\UserPicker.razor.cs(51,39,51,57): error CS0115: 'UserPicker.OnInitializedAsync()': no suitable method found to override

EventHub.Admin.Web\Pages\AttendeeDetail.razor.cs(37,39,37,57): error CS0115: 'AttendeeDetail.OnInitializedAsync()': no suitable method found to override

Can you plz help to fix these issues.
I am using visual studio 2022

--- a/src/EventHub.DbMigrator/EventHubDbMigratorModule.cs
+++ b/src/EventHub.DbMigrator/EventHubDbMigratorModule.cs
@@ -32,7 +32,7 @@ namespace EventHub.DbMigrator
                     IsExtendable = false
                 });
             });
-        }
 #endif
+        }
     }
 }

